extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {AMPDOC_SINGLETON_NAME} from '../../../src/service/ampdoc-impl';
```

Comment:
Maybe we should move this elsewhere, with only this in the file? That way there are no transitive imports, and we don't need the presubmit whitelist?

Reply:
I agreed! Any idea where to move it to. I wish there's a src/const.js file.

Reply:
That sounds fine to me. Or `src/enums.js`